### PR TITLE
feat: AWS Connectors Automation

### PR DESCRIPTION
**Pull request by felipecosta09**
## Cloud One Service

[X] **Common**

[ ] **Workload Security**

[ ] **Application Security**

[ ] **Network Security**

[ ] **File Storage Security**

[ ] **Container Security**

[ ] **Conformity**

[ ] **Open Source Security**

## Proposed Changes
  - Add project to automate connector creation

### CODE REVIEW

File: Common/Cloud-Account/aws-tf-connectors-automation/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Cloud One Connector Automation
+
+ This automation is intended to streamline the process of setting up the Cloud One common connector for AWS, workload security connector for AWS and the AWS cloudtrail integration.
```
Comment (JustinDPerkins) [inline Common/Cloud-Account/aws-tf-connectors-automation/README.md:3]: Could we link this to the C1 Doc page that explains the connector

Comment (JustinDPerkins) [reply Common/Cloud-Account/aws-tf-connectors-automation/README.md:3]: if that exists of course

File: Common/Cloud-Account/aws-tf-connectors-automation/README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+After cloning this repo, go to the `connectors.tfvars` and input the required information to deploy the connectors to Cloud One.
+
+ Then you can execute the following:
+
+   ```
+   terraform init
+   terraform plan -var-file="connectors.tfvars"
+   terraform apply -auto-approve -var-file="connectors.tfvars"
+   ```
+
```
Comment (JustinDPerkins) [inline Common/Cloud-Account/aws-tf-connectors-automation/README.md:22]: Maybe make this a unique section like ###How to Verify

File: Common/Cloud-Account/aws-tf-connectors-automation/README.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+All the connectors should be deployed, to check the status on the cloudtrail integration, go to the cloudformation page in the desired AWS account to check the status, once completed, you should be able to receive the events on your Vision One account.
+
```
Comment (JustinDPerkins) [inline Common/Cloud-Account/aws-tf-connectors-automation/README.md:24]: Same here: how to remove

File: Common/Cloud-Account/aws-tf-connectors-automation/aws_connectors.tf [hunk 4]
```diff
@@ -0,0 +1,78 @@
+# Wait time to sync the IAM
+resource "time_sleep" "wait_20_seconds" {
+  create_duration = "20s"
+}
+
+locals {
+  c1-region = var.c1-region
+  c1-api-key = var.c1-api-key
```
Comment (JustinDPerkins) [inline Common/Cloud-Account/aws-tf-connectors-automation/aws_connectors.tf:8]: If we require a c1 api key it should be noted in the readme with how to apply.

File: Common/Cloud-Account/aws-tf-connectors-automation/aws_connectors.tf [hunk 5]
```diff
@@ -0,0 +1,78 @@
+# Wait time to sync the IAM
+resource "time_sleep" "wait_20_seconds" {
+  create_duration = "20s"
+}
+
+locals {
+  c1-region = var.c1-region
```
Comment (JustinDPerkins) [inline Common/Cloud-Account/aws-tf-connectors-automation/aws_connectors.tf:7]: Add to readme as well to obtain this info

File: Common/Cloud-Account/aws-tf-connectors-automation/connectors.tfvars [hunk 6]
```diff
@@ -0,0 +1,7 @@
+region = "us-east-1"
```
Comment (JustinDPerkins) [inline Common/Cloud-Account/aws-tf-connectors-automation/connectors.tfvars:1]: I would explain that this file needs to be updated with the unique values for deployment to work.

**Review by JustinDPerkins — CHANGES_REQUESTED** [review]

**Review by atiradocc — APPROVED** [review]

**Pull request closed without merging**